src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -706,6 +709,9 @@ void Convert::caesuraFromMEI(engraving::Breath* breath, const libmei::Caesura& m
         }
     }
 
+    // caesura doesn't support @place, so we set default placement
+    breath->setPlacement(breath->propertyDefault(engraving::Pid::PLACEMENT).value<engraving::PlacementV>());
```

Comment:
When a `Breath` is constructed, all its properties should be at their default state, right? So does the necessity of this addition mean that the constructor is not properly doing that, or is the placement first being set elsewhere?

Reply:
I finally figured this out: the default placement is baseed on the track, but we create a breath without a track, so we have to do the placement after it received it's track.

Reply:
Sorry.